wicket-experimental/wicket-atmosphere/src/main/java/org/apache/wicket/atmosphere/AtmosphereBehavior.java

```diff
@@ -16,11 +16,7 @@
  */
 package org.apache.wicket.atmosphere;
 
-import org.apache.wicket.Application;
-import org.apache.wicket.Component;
-import org.apache.wicket.MetaDataKey;
-import org.apache.wicket.Page;
-import org.apache.wicket.WicketRuntimeException;
+import org.apache.wicket.*;
```

Comment:
As far as I know, wicket code convention doesn't allow to use imports with wildcard.